interface/docker-based-rock-ons/immich.rst

```diff
@@ -0,0 +1,352 @@
+If you wanted to have a central repository of photos/videos that is shared between different users, 
+`you will need to wait <https://github.com/immich-app/immich/issues/12614>`_ until it's implemented.
+
+Until then, here are some alternatives:
+
+#. Create a single user account that you share with other people: all people and devices will have access to the same library with all people/facial recognition data (and other metadata), and they will all be able to upload their own files to the library. Duplicates are detected on the whole library.
+#. `Use partner sharing <https://docs.immich.app/features/partner-sharing/>`_: other users will be able to view and download files (and their metadata) from your library, but won't also share the people/facial recognition data. You won't see other users' libraries automatically - they have to explicitly share back. Other users can't upload their files to your library. Duplicates are detected only on your library. Smart search and map features won't be shared.
+#. Create user accounts for each family member and one additional account with the external library, then using the partner sharing feature, share the external library with other users.
+#. Set-up the same external libraries for each user: causes unnecessary file metadata duplication.
+
+Here is how you can import your files into Immich.
+
+Prerequisite is that you have made your share available to Immich. See: :ref:`immich_mounting_external_library`. 
+Also note that every external library must have an owner. If you haven't created your user account(s), do so now. See: :ref:`immich_new_user`
+
+Click on your profile image on the top-right corner of the page, then click on **Administration**.
+                                   
+.. image:: /images/interface/docker-based-rock-ons/immich_administration.png
+   :width: 40%
+   :align: center
+
+ 
+Click on the **External Libraries** item in the left-side menu.
+                                   
+.. image:: /images/interface/docker-based-rock-ons/immich_external_libraries.png
+   :width: 40%
+   :align: center
+
+
+Click on the **Create library** link below your profile image on the top-right corner of the page.
+                                   
+.. image:: /images/interface/docker-based-rock-ons/immich_create_library.png
+   :width: 40%
+   :align: center
+
+
+Choose a user that will be the owner of the external library. This means that 
+all files from the external library will be imported to the selected user's library. 
+The user may choose to 
+`share their library with other users with some limitations <https://docs.immich.app/features/partner-sharing/>`_.
+
+Click on the **Create** Button.
+
+.. image:: /images/interface/docker-based-rock-ons/immich_external_library_owner.png
+   :width: 50%
+   :align: center
+
+Input the path to the external library that you've defined at :ref:`immich_mounting_external_library`. 
+E.g. ``/mnt2/MyPrivatePhotoCollection``, ``/data/test1``...
```

Comment:
```suggestion
E.g. ``/data/test1``...
```
@kanecko I think this is less confusing as I think only the internal reference will be shown at this point.